### PR TITLE
update copyright

### PR DESCRIPTION
**Pull request by robbecker-wf**
Update the copyright info to 2015 Workiva Inc.

### CODE REVIEW

**Comment by stephenbush-wf** [comment]
Looks like you might have missed a couple, 
```
wf-uicomponents/scripts/build/start.frag
wf-uicomponents/src/awesome_map/BoundaryTypes.js
```

**Comment by stephenbush-wf** [comment]
Also a few places where it says WebFilings!  Could roll those in

**Comment by robbecker-wf** [comment]
Updated WebFilings to Workiva

File: README.md [hunk 1]
```diff
@@ -37,8 +37,6 @@ There are three interceptors provided out of the box:
 - **ScaleInterceptor**: Restricts zooming of content to specified scale limits.
 - **SwipeInterceptor**: Animates a swipe when swipe events are transformed.
 
-[Demo](https://webfilings.box.com/shared/static/4aj3gfi3u2hmkqueashm.mov)
```
Comment (robbecker-wf) [inline README.md]: Removed this demo link since it no longer works.

**Comment by stephenbush-wf** [comment]
Hm.  Looks like there's one more old copyright, in `wf-uicomponents/examples/vendor/wf-uicomponents.min.js` which i thought was a generated, minified file that would update/go away after grunt.  But nope.  Looks like its an actual, non-built committed project file.  Wtf?

Also, a couple more WebFilings in 
```
wf-uicomponents/docs/README.md
wf-uicomponents/docs/api/index.html
```

_(Sorry >__>)_

**Pull request closed without merging**